src/prototype_room_fight.js

```diff
@@ -1,12 +1,12 @@
-Room.prototype.getFriends = function(object) {
+Room.prototype.getFriends = function() {
```

Comment:
it's for keep object argument and allow to use the function for check if object is friend instead of find for first hostile creep.